metron-interface/metron-alerts/src/app/alerts/alerts-list/table-view/table-view.component.ts

```diff
@@ -187,9 +191,12 @@ export class TableViewComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   formatValue(column: ColumnMetadata, returnValue: string) {
+    this.localTime = this.timezoneConfigService.getTimezoneConfig();
     try {
-      if (column.name.endsWith(':ts') || column.name.endsWith('timestamp')) {
-        returnValue = new Date(parseInt(returnValue, 10)).toISOString().replace('T', ' ').slice(0, 19);
+      if ((column.name.endsWith(':ts') || column.name.endsWith('timestamp')) && this.localTime === true) {
+        returnValue = moment.utc(returnValue).local().format('YYYY-MM-DD H:mm:ss');
```

Comment:
One small Q here - can you confirm how this will work when the user's browser and the host server are in 2 different time zones? I'm assuming that since this is Angular client-side code it will the user's location, not the location of the Metron installation.

Reply:
That's correct, it references the user's location. Specifically, it references the location in their computer's OS time settings.